v2/googlecloud-to-googlecloud/src/main/java/com/google/cloud/teleport/v2/utils/BigtableUtils.java

```diff
@@ -0,0 +1,305 @@
+package com.google.cloud.teleport.v2.utils;
+
+import com.google.cloud.bigtable.data.v2.models.ChangeStreamMutation;
+import com.google.cloud.bigtable.data.v2.models.ChangeStreamMutation.MutationType;
+import com.google.cloud.bigtable.data.v2.models.DeleteCells;
+import com.google.cloud.bigtable.data.v2.models.DeleteFamily;
+import com.google.cloud.bigtable.data.v2.models.Entry;
+import com.google.cloud.bigtable.data.v2.models.SetCell;
+import com.google.cloud.teleport.bigtable.BigtableRow;
+import com.google.cloud.teleport.bigtable.ChangelogEntry;
+import com.google.cloud.teleport.v2.templates.bigtablechangestreamstogcs.model.ChangelogColumns;
+import java.nio.ByteBuffer;
+import java.nio.charset.Charset;
+import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.concurrent.atomic.AtomicLong;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * A set of helper functions and classes for Bigtable.
+ */
+public class BigtableUtils {
+
+  public static String bigtableRowColumnFamilyName = "changelog";
+
+  public static String bigtableRowKeyDelimiter = "#";
+
+  public static String columnPattern = "^[^:]+:.*$";
+
+  private static final Logger LOG = LoggerFactory.getLogger(BigtableUtils.class);
+
+  private static Boolean isValidEntry(String familyName, String qualifierName,
+      HashSet<String> ignoreColumns, HashSet<String> ignoreColumnFamilies) {
+    if (familyName == null) {
+      return true;
+    }
+
+    if (ignoreColumnFamilies.contains(familyName)) {
+      return false;
+    }
+
+    String columnFamilyAndQualifier = familyName + ":" + qualifierName;
```

Comment:
I think we need to check if qualifierName==null or not here. For some mutations, column name is not present

Reply:
Good catch, I will change this to an empty string if it is null